src/lgtm/scripts/run/grafana.sh

```
@@ -0,0 +1,5 @@
+export GF_AUTH_ANONYMOUS_ENABLED=true
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)